cadc-log/src/main/java/ca/nrc/cadc/log/WebServiceLogInfo.java

```diff
@@ -135,14 +130,50 @@ public String start() {
      */
     public String end() {
         this.success = userSuccess;
-        return "END: " + doit();
+        return "{" + getPreamble() + "\"phase\":\"end\"," + doit() + "}";
     }
 
     String doit() {
         StringBuilder sb = new StringBuilder();
-        sb.append("{");
         populate(sb, this.getClass());
-        sb.append("}");
+        return sb.toString();
+    }
+
+    private String getTimestamp() {
+        DateFormat format = DateUtil.getDateFormat(DateUtil.ISO_DATE_FORMAT,  DateUtil.UTC);
+        Date date = new Date(System.currentTimeMillis());
+        return "\"@timestamp\":\"" + format.format(date) + "\"";
+    }
+
+    private String getServiceName() {
+        return "\"service\":{\"name\":\"" + serviceName + "\"}";
+    }
+
+    private String getThreadName() {
+        return "\"thread\":{\"name\":\"" + Thread.currentThread().getName() + "\"}";
+    }
+
+    private String getLoglevel() {
+        return "\"log\":{\"level\":\"info\"}";
```

Comment:
Shouldn't we be returning the actual current log level?

Reply:
This class is used only for info level logging. You could add a logging level variable to the class, default it to info, and add a function to set the log level. But it would, or should, always be at the info level.